### DISCUSSION TITLE
Shouldn't the documentation show how to install the command line tools on a Mac?

### DISCUSSION THREAD

**Discussion by alamo42** [pulsar-edit/.github#170] [Documentation & Website]
Today I installed pulsar on my M1 Mac, tried to run `pulsar` on the terminal and I was welcomed with a "Command not found" kind of error. After some time researching I tried to run the `pulsar.sh` on the .app's Resources and received a more general error from the apple system.

Anyways, just when I felt I should be opening an issue for the errors above I opened the Command Palette on pulsar and inputed shell, as in VSCode, and I found the option to install the shell program. 

It was there the whole time!

I didn't find in the documentation that way of installing the cmd tools, so I was thinking of issuing a PR on Hacking the Core. Like adding some tip message to avoid further confusion by other future users. What do you think?

**Comment by Daeraxa** [comment]
I was sure we had this before and I was right, we do, but it is commented out (this was from a while ago before we had this working at all.)

We haven't had much movement on the linked issue so it might be prudent to uncomment this for mac at least.

```
<!--TODO: Check if this is actually still going to be used or if will be generic - see https://github.com/pulsar-edit/pulsar/issues/187
<span class="platform-mac">The Pulsar menu bar has a command named
"Install Shell Commands" which installs the `atom` and `apm` commands
[if Pulsar wasn't able to install them itself](/installing-atom/#installing-atom-on-mac)
.</span><span class="platform-windows platform-linux">The `atom` and `apm`
commands are installed automatically as a part of Pulsar's
[installation process](/sections/installing-atom/).</span>
-->
```

**Reply by alamo42** [reply]
I will submit the PR as soon as I can then. Thank you!
